src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1010,6 +1012,33 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   else
     options.set_option("simplify", true);
 
+  // use assumptions instead of assertions?
+  if(cmdline.isset("assert-to-assume"))
```

Comment:
I think this is included in `PARSE_OPTIONS_GOTO_CHECK(cmdline, options);` which is just a few lines lower.